### PR TITLE
Jack support

### PR DESCRIPTION
**Pull request by secondsun**
@mosabua Initial / experimental support for jack here.

YouTube Demo : https://youtu.be/0b3wGdVulIs

### CODE REVIEW

**Comment by secondsun** [comment]
This is the WIP for https://github.com/simpligility/android-maven-plugin/issues/536

**Comment by mosabua** [comment]
Wdyt @secondsun .. merge to master or feature branch? I am kind of thinking master ... we wont cut a release for a while and then we have it right there.

**Comment by secondsun** [comment]
@mosabua I'm fine with master if you are

File: pom.xml [hunk 1]
```diff
@@ -886,6 +897,13 @@
           <relaxed>false</relaxed>
         </configuration>
       </plugin>
+      <plugin>
```
Comment (mosabua) [inline pom.xml]: This should be removed since a checkstyle enforcement is already part of the project.

**Comment by Shusshu** [comment]
Good job!

Should the proguardMojo also have the following code?

    if ( getJack().isEnabled() ) 
    {
        //proguard is handled by Jack
        return;
    }

**Comment by mosabua** [comment]
Probably @Shusshu .. this is in master now. You should commit the fix ;-)

**Comment by Shusshu** [comment]
https://github.com/simpligility/android-maven-plugin/pull/744 ;) via github

**Comment by mosabua** [comment]
Merged already .. haha! Going to make a changelog entry next ..

**Comment by natami** [comment]
Any news on this?

**Pull request merged**